docs/reference/release-notes/v2.29/index.md

```diff
@@ -0,0 +1,133 @@
+---
+date: "2020-12-15"
+version: "2.29.0"
+---
+
+# [v2.29](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.29.0-next.0...gatsby@2.29.0) (December 2020 #2)
+
+---
+
+Welcome to `gatsby@2.29.0` release (December 2020 #2)
+
+Key highlights of this release:
+
+- [Query on Demand](#query-on-demand) - improves `gatsby develop` bootup time
+- [Lazy Images](#lazy-images) - only do image processing when a page gets reqested
+- [Improvements to our CLI](#improvements-to-our-cli) - improved `create-gatsby` & new `plugin` command
+- [Experimental: Parallel data sourcing](#experimental-parallel-data-sourcing) - run source plugins in parallel to speedup sourcing on sites with multiple source plugins
+
+Other notable changes:
+
+- [Performance improvements](#performance-improvements)
+- [Slugify option for File System Route API](#slugify-option-for-file-system-route-api)
+- [gatsby-image codemod](#gatsby-image-codemod)
+- [Notable bugfixes](#notable-bugfixes)
+
+Sneak peak to next releases:
+
+- TODO
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know
+if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](../v2.28/index.md)<br>
+[Full changelog](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.29.0-next.0...gatsby@2.29.0)
+
+## Query on Demand
+
+Starting with v2.29, 10% of our users are automatically opt-in to this feature. We've first shipped this feature behind a flag in [v2.27](../v2.27/index.md#experimental-queries-on-demand) and feel confident now that more people can try it out. Opt-in users will receive a notice in their terminal about that opt-in behavior and a hint on how to turn it off (in case it disturbs your workflow). As a recap of what Query on Demand will improve:
+
+> Gatsby will run queries for pages as they're requested by a browser. Think of it like lazily loading the data your pages need, when they need it! This avoids having to wait for slower queries (like image processing) if you're editing an unrelated part of a site. What this means for you: faster local development experience, up to 2x faster in many cases!
+
+By slowly rolling out this feature to more and more users we'll be getting closer to a GA (general availability) release that will benefit all users of Gatsby. Please give us your feedback in the [umbrella discussion](https://github.com/gatsbyjs/gatsby/discussions/27620).
+
+If you want to turn it on/off, you can set the corresponding flag inside `gatsby-config.js`:
+
+```js
+// In your gatsby-config.js
+module.exports = {
+  flags: {
+    QUERY_ON_DEMAND: true,
```

Comment:
Should this add comment to set it to `false` for users auto opted-in that found problems and don't want to use it?